content/css/jenkins.css

```diff
@@ -21,6 +21,10 @@ h6[id]:before {
   opacity: 0;
 }
 
+img {
+  max-width: 100%;
+}
+
```

Comment:
I've added this because in cases like this where the animated png is very big, the image would be rendered at full width and grow out of it's parent element, making the browser show a horizontal scrollbar. This limits the size of all images to that of their container tag.